src/battle_cam.c

```diff
@@ -0,0 +1,2562 @@
+#include "common.h"
+#include "script_api/battle.h"
+
+#define LERP(a, b, alpha) ((a) * (alpha) + (b) * (1.0f-(alpha)))
```

Comment:
Given this computes alpha twice could we maybe turn it into a static function

Reply:
I don't want us to add any static functions